scripts/linux/agent_diagnostics.sh

```diff
@@ -123,9 +126,9 @@ function info_out() {
     OS=$( grep release /etc/redhat-release )
   fi
   SERVICE="jcagent"
-  STATUS=$( service ${SERVICE} status 2> /dev/null )
+  STATUS=$( service "${SERVICE}" status 2> /dev/null )
   if [[ -z "${STATUS}" ]]; then
-    STATUS=$( echo "PID  PATH" ; ps ax | grep [j]cagent | awk '{print $1" "$5}')
+    STATUS=$( echo "PID  PATH" ; pgrep -f -a "jcagent" | awk '{print $1" "$5}')
```

Comment:
Didn't know this would silence SC2009 *and* work that same. Also, we need to grep for "jumpcloud-agent" now. The process name is no longer `jcagent`, and I don't know when it was.

Reply:
Good catch, thanks! I had this fixed on my VM, but forgot to copy it back before I pushed. Fixed.